Lib/fontTools/ttLib/tables/O_S_2f_2.py

```diff
@@ -300,6 +323,37 @@ def recalcUnicodeRanges(self, ttFont, pruneOnly=False):
 		self.setUnicodeRanges(bits)
 		return bits
 
+	def recalcCodePageRanges(self, ttFont, drop=[0x0000]):
+		""" 
+		Function to recalculate OS/2 fsSelection CodePageRanges
+		Drop 0x0000 by default, because most fonts have the character NULL not encoded
+		"""
+		from fontTools.unicodedata.codepages import OS2_CODEPAGE_RANGES, CODEPAGERANGES_ENCODINGS
+		for table in ttFont['cmap'].tables:
+			# if symbol font (cmap = platform 3 encoding 0 ) 
+			# set bit 31 only
+			if table.platformID == 3 and table.platEncID == 0:
+				bits = {31}
+				self.setCodePageRanges(bits)
+				return bits
+		unicodes = set()
+		for table in ttFont['cmap'].tables:
+			if table.isUnicode():
+				unicodes.update(table.cmap.keys())
+		bits = set()
+		for bit in OS2_CODEPAGE_RANGES:
+			codepage_name = OS2_CODEPAGE_RANGES[bit]
+			if codepage_name is None:
+				continue
+			codepage_encoding = CODEPAGERANGES_ENCODINGS[codepage_name]
+			if drop:
+				for enc in drop:
+					codepage_encoding.discard(enc)
+			if set() == codepage_encoding.difference(unicodes):
```

Comment:
`if unicodes.issuperset(codepage_encoding)`

Reply:
I have changed it, thanks :)